Add section on Wrangler env variables (#3178)
cloudflare/cloudflare-docs: products/workers/src/content/cli-wrangler/commands.md

--------------------------------

## Environment variables

> Wrangler also supports any `wrangler.toml` keys passed in as environment variables. This works by passing in `CF_` + any toml key uppercased. So you can pass them in like:

Reviewer: ```suggestion
Wrangler supports any `wrangler.toml` keys passed in as environment variables. This works by passing in `CF_` + any uppercased TOML key. For example:
```
